codegen/apt/src/main/java/io/helidon/codegen/apt/AptContextImpl.java

```diff
@@ -101,7 +101,7 @@ private static Optional<ModuleInfo> findModule(Filer filer) {
             try (InputStream in = resource.openInputStream()) {
                 return Optional.of(ModuleInfoSourceParser.parse(in));
             }
-        } catch (IOException ignored) {
+        } catch (Exception ignored) {
```

Comment:
Should we consider catching `IOException` and `IllegalArgumentException` instead?

Reply:
I think catching `Exception` is sufficient here, as it shows that there was "a" problem getting the `module-info.java`. As we have some fallback mechanism here, and we do not require it, it should be fine.
It may be interesting to find an alternative for Eclipse IDE (is there "any" way to get the module info?), because we use the module info to get the module name, which is used in some generated types (the worst case here is that the used name will be `unnamed/package-name`) for documentation purposes.

Reply:
I double-checked the ECJ implementation and it only handles `javax.tools.StandardLocation` values that point to generated source code. Which means, we can't do anything here. We would need to adapt the ECJ and implement the missing cases we need.